config/routes.rb

```diff
@@ -298,6 +299,7 @@
     resources :delayed_jobs
     resources :epp_logs
     resources :repp_logs
+    resources :bounced_mail_addresses
```

Comment:
How about add `only: %i[index show destroy]` so we are explicitly setting there will be no updates/creates?

Reply:
Good catch! Going to change it for sure